send.sh

```diff
@@ -12,13 +12,13 @@ echo -e "[Webhook]: Sending webhook to Discord...\\n";
 
 AVATAR="https://github.com/actions.png"
 
-case $1 in
-  "Success" )
+case ${1,,} in
```

Comment:
Could you add a comment to some form of GNU docs that explains how this works?

Reply:
From [the Bash manual](https://www.gnu.org/software/bash/manual/bash.html):

> ${parameter^pattern}
> ${parameter^^pattern}
> ${parameter,pattern}
> ${parameter,,pattern}
> This expansion modifies the case of alphabetic characters in parameter. The pattern is expanded to produce a pattern just as in filename expansion. Each character in the expanded value of parameter is tested against pattern, and, if it matches the pattern, its case is converted. The pattern should not attempt to match more than one character. The ‘^’ operator converts lowercase letters matching pattern to uppercase; the ‘,’ operator converts matching uppercase letters to lowercase. The ‘^^’ and ‘,,’ expansions convert each matched character in the expanded value; the ‘^’ and ‘,’ expansions match and convert only the first character in the expanded value. If pattern is omitted, it is treated like a ‘?’, which matches every character. If parameter is ‘@’ or ‘*’, the case modification operation is applied to each positional parameter in turn, and the expansion is the resultant list. If parameter is an array variable subscripted with ‘@’ or ‘*’, the case modification operation is applied to each member of the array in turn, and the expansion is the resultant list.

Do you want me to add a comment indicating this in the script?

Reply:
Sorry for the delay, yeah, could we add link?

Something like this?
```bash
# More info: https://www.gnu.org/software/bash/manual/bash.html#Shell-Parameter-Expansion
case ${1,,} in
```